umapi_client/connection.py

```diff
@@ -310,6 +313,20 @@ def execute_multiple(self, actions, immediate=True):
         for a in actions:
             if len(a.commands) == 0:
                 if self.logger: self.logger.warning("Sending action with no commands: %s", a.frame)
+            # throttling part 1a: split commands with group assignments > self.throttle_groups
+            # if an action gets more than 10 commands per verb/group type combination, then it will
+            # be split below
+            for i, c in enumerate(a.commands):
+                for verb in ['add', 'remove']:
```

Comment:
Are these the only verbs that take group lists?  And when we add support for Group actions (rather than User actions) to these verbs also take group lists?  Or are they user lists and are they throttled?  Make sure this is the right test.

It probably makes more sense to add a data structure in the functional module (which is where this code is going) which says which commands in which action types actually takes lists of groups as their argument.  That way this test can just be data driven rather than having the constants in the code.